graphql_compiler/tests/test_helpers.py

```diff
@@ -281,14 +284,35 @@ def get_schema():
     return schema
 
 
+def get_schema_graph():
+    """Get a SchemaGraph object for testing."""
+    return SchemaGraph(get_schema_data())
+
+
+def get_schema_data():
+    """Return the OrientDB schema data for testing."""
+    with open(SCHEMA_DATA_JSON_FILE) as infile:
```

Comment:
This opens a file relative to the current working directory, which may change and may break the tests as a result. Load the file relative to the current file's location instead, that way the test doesn't care about the current working directory.